pkgs/top-level/all-packages.nix

```diff
@@ -2820,10 +2820,6 @@ with pkgs;
 
   sonobuoy = callPackage ../applications/networking/cluster/sonobuoy { };
 
-  strawberry-qt6 = qt6Packages.callPackage ../applications/audio/strawberry { };
```

Comment:
Looks like `strawberry-qt6` is gone without an alias being added in pkgs/top-level/aliases.nix?